notebooks/00_core.ipynb

```diff
@@ -27,7 +27,8 @@
     "# export\n",
```

Comment:
<p>why did you add subprocess?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Ah yes, this is an artifact from a hack I was trying to implement that would skip the download if the file already existed. I'll remove it for now</p>